src/main/java/com/codeborne/selenide/SelenideConfig.java

```diff
@@ -33,14 +30,12 @@
   private boolean holdBrowserOpen = Boolean.getBoolean("selenide.holdBrowserOpen");
   private boolean reopenBrowserOnFail = Boolean.parseBoolean(System.getProperty("selenide.reopenBrowserOnFail", "true"));
   private long closeBrowserTimeoutMs = Long.parseLong(System.getProperty("selenide.closeBrowserTimeout", "5000"));
-  private boolean clickViaJs = Boolean.parseBoolean(System.getProperty("selenide.clickViaJs",
-    System.getProperty("selenide.click-via-js", "false")));
+  private boolean clickViaJs = Boolean.parseBoolean(System.getProperty("selenide.clickViaJs", "false"));
   private boolean captureJavascriptErrors = Boolean.parseBoolean(System.getProperty("selenide.captureJavascriptErrors", "true"));
   private boolean screenshots = Boolean.parseBoolean(System.getProperty("selenide.screenshots", "true"));
 
   private boolean savePageSource = Boolean.parseBoolean(System.getProperty("selenide.savePageSource", "true"));
-  private String reportsFolder = System.getProperty("selenide.reportsFolder",
-    System.getProperty("selenide.reports", "build/reports/tests"));
```

Comment:
Since you have dropped the support for `selenide.reports` system variable in favor of `selenide.reportsFolder`, you should probably update the official documentation: https://selenide.org/documentation/screenshots.html

Reply:
It seems that it's already fixed? I see the following text now:

> Yes. You can use property `-Dselenide.reportsFolder=test-result/reports` to set any directory to store screenshots to. For v4.x use `-Dselenide.reports=test-result/reports`

Reply:
yes, it was fixed in scope of issue #894 by @BorisOsipov